src/components/TableBody.js

```diff
@@ -280,8 +280,8 @@ class TableBody extends React.Component {
               options={options}
               colIndex={0}
               rowIndex={0}>
-              <Typography variant="subtitle1" className={classes.emptyTitle}>
-                {options.textLabels.body.noMatch}
+              <Typography variant="body1" className={classes.emptyTitle}>
+                <strong>{options.textLabels.body.noMatch}</strong>
```

Comment:
Lets lose the `strong` tags to keep better backwards compatibility with the appearance, but otherwise looks good.